tests/python/test_trace2.c

```diff
@@ -6,7 +6,7 @@ struct Counters { u64 stat1; };
 BPF_TABLE("hash", struct Ptr, struct Counters, stats, 1024);
 
 int count_sched(struct pt_regs *ctx) {
-  struct Ptr key = {.ptr=ctx->bx};
+  struct Ptr key = {.ptr = ctx->PT_REGS_PARM1};
```

Comment:
I can't honestly recall why `ctx->bx` got put in there. This is one of the older test cases. Best guess is that I needed a non-zero value to serve as a key, but the specifics here aren't really important. Probably rather than using the arch-independent macro, the function should be changed to use `u64 prev` in the count_sched prototype, rather than directly pulling from ctx-> directly.

Something like:
```
int count_sched(struct pt_regs *ctx, u64 prev) {
  struct Ptr Key = {.ptr = prev};
```